### PR TITLE
updated signing instructions and build script

### PR DESCRIPTION
**Pull request by marionleborgne**
small update of the signing instructions & build script

cc @lscheinkman @brev

### CODE REVIEW

File: unicorn/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "x64"
   ],
   "scripts": {
-    "build": "npm install && UNICORN_TARGET=desktop npm run pack  && electron-packager . Unicorn --prune --ignore=.git --ignore=node_modules/electron-*  --platform=darwin --arch=x64 --version=0.32.3 --overwrite --icon=assets/unicorn.icns",
+    "build": "npm install && UNICORN_TARGET=desktop npm run pack  && electron-packager . Unicorn --prune --ignore=.git --ignore=node_modules/electron-*  --platform=darwin --arch=x64 --version=0.32.3 --overwrite --icon=assets/unicorn.icns --sign='Developer ID Application: Numenta, Inc. (2589Z673MU)'",
```
Comment (brev) [inline unicorn/package.json]: Hi @marionleborgne is this our actual key string, or just an example string? (Not sure if it's supposed to be secret or not?)

Comment (brev) [inline unicorn/package.json]: Hi @marionleborgne, It's usually better not to hard-code any settings -- how about something like the following?

```shell
export UNICORN_CERT_DEV_APPLE="Developer ID Application: Numenta, Inc. (ABCDEFGHIJK)"
npm run build -- --sign=$UNICORN_CERT_DEV_APPLE
```

Details on how the `npm` `run-scripts` args work: https://docs.npmjs.com/cli/run-script

**Comment by brev** [comment]
@marionleborgne I added a few small notes, but otherwise :+1: looking great.

File: unicorn/package.json (hunk 1 above)
Comment (marionleborgne) [inline unicorn/package.json]: Somehow, I completely missed your comments, sorry! I generated another ID. This one is obsolete. I will use the env variable as you mentioned.

Comment (marionleborgne) [inline unicorn/package.json]: The build script stay as is (I didn't add the `--sign` flag) However, I updated the readme instructions to explain how to sign the app. Thanks for the link, that was useful. Let me know if I can merge this branch.

Comment (marionleborgne) [inline unicorn/package.json]: Hum. Actually @brev, I spoke too fast: `npm run build -- --sign=$UNICORN_CERT_DEV_APPLE` does not add the `--sign` flag correctly. This is what it gives me:

```
UNICORN_TARGET=desktop npm run clean && npm install && npm run prepare && npm run prepare:python && npm run electron:packager "--sign=Developer" "ID" "Application:" "Numenta," "Inc." "(2589Z673MU)"
```


I might have to include `--sign=$UNICORN_CERT_DEV_APPLE` directly in the `build` option of `package.json`.

Comment (marionleborgne) [inline unicorn/package.json]: And re-update the README instructions :/

**Comment by brev** [comment]
@marionleborgne sorry about the npm  stuff not working, we can look at it later. don't let this hold up the merge at all! thanks

**Comment by lscheinkman** [comment]
@marionleborgne Please remove the files you did not change from the PR to avoid merge conflicts. From what I understood this PR should only change the `unicorn/README.md` file

**Comment by brev** [comment]
I can't tell, did I accidentally merge something weird into here? Sorry if so.

**Comment by lscheinkman** [comment]
@brev This PR will conflict with #388 as is. To avoid that conflict it should only contain the `README.md` file and leave the other files (`Chart.jsx` and `app.js`) out of the PR

**Comment by marionleborgne** [comment]
@lscheinkman @brev Irrelevant files are now out of this PR. Only `README.md` and `package.json` have been modified.

**Comment by lscheinkman** [comment]
:+1: LGTM

**Pull request merged**